estimate-web/src/components/UserStoryCommentEditor.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from "react";
+import { PropTypes } from "prop-types";
+
+import {
+    TextField, Persona, PersonaSize, PrimaryButton, Spinner
```

Comment:
In most files I use one import per line once it reaches a certain amount of lines. This is due to ESLint enforcement rules, wonder if it's different for you

Reply:
It doesn't complain for me, but I've changed it now. It was probably still below the max. line length.